apps/docs/src/app/core/component-docs/moment-datetime-adapter/moment-datetime-adapter-docs.component.html

```diff
@@ -0,0 +1,61 @@
+<fd-docs-section-title
+    [id]="'simple'"
+    [componentName]="'momentDatetimeAdapter'"
```

Comment:
```suggestion
    componentName="momentDatetimeAdapter"
```

Reply:
please change the `componentName` to `moment-datetime-adapter` or else the anchor links won't work

Reply:
there was typo